### PR TITLE
ddl: refine GetDDLInfo function

### PR DESCRIPTION
**Pull request by xiongjiwei**
### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: executor/executor_test.go [hunk 1]
```diff
@@ -5623,9 +5623,10 @@ func TestAdmin(t *testing.T) {
 	require.NoError(t, err)
 	row = req.GetRow(0)
 	require.Equal(t, 6, row.Len())
-	txn, err := store.Begin()
+	sess := testkit.NewTestKit(t, store).Session()
+	_, err = sess.Execute(context.Background(), "begin")
```
Comment (hawkingrei) [inline executor/executor_test.go:5627]: ```tk.MustExec```?

In history, We forget to close the record set to cause a lot of problems.

File: executor/executor_test.go [hunk 2]
```diff
@@ -5623,9 +5623,10 @@ func TestAdmin(t *testing.T) {
...
 	require.NoError(t, err)
-	ddlInfo, err := ddl.GetDDLInfo(txn)
+	ddlInfo, err := ddl.GetDDLInfo(sess)
 	require.NoError(t, err)
 	require.Equal(t, ddlInfo.SchemaVer, row.GetInt64(0))
 	// TODO: Pass this test.
```
Comment (hawkingrei) [inline executor/executor_test.go:5632]: remove this comment?

**Comment by hawkingrei** [comment]
Other parts PTAL

File: executor/executor_test.go (hunk 2 above)
Comment (xiongjiwei) [reply executor/executor_test.go:5632]: looks like it is a very old test. I will keep it here and think maybe we can remove it

**Review by hawkingrei — APPROVED** [review]

**Review by xhebox — APPROVED** [review]

**Pull request merged**